mlflow/utils/_capture_modules.py

```diff
@@ -110,10 +111,23 @@ def parse_args():
     parser.add_argument("--output-file", required=True)
     parser.add_argument("--sys-path", required=True)
     parser.add_argument("--module-to-throw", required=False)
+    parser.add_argument("--error-file", required=True)
     return parser.parse_args()
 
 
-def store_imported_modules(cap_cm, model_path, flavor, output_file):
+def _get_stacktrace(error):
```

Comment:
We might want to move this to some utils file if it's commonly used

Reply:
Good idea. This exactly implementation exists within `SpanEvent` in tracing too.

Reply:
+1